README.rst

```diff
@@ -203,91 +220,76 @@ https://training.golioth.io/docs/zephyr-training
    Do not clone this repo using git. Zephyr's ``west`` meta-tool should be used
    to set up your local workspace.
 
-Create a Python virtual environment (recommended)
-=================================================
+Install the Python virtual environment (recommended)
+====================================================
 
 .. code-block:: shell
 
    cd ~
    mkdir golioth-reference-design-greenhouse
-   python -m venv golioth-reference-design-greenhouse/.venv
+   python -m venv golioth-reference-greenhouse/.venv
    source golioth-reference-design-greenhouse/.venv/bin/activate
-
-Install ``west`` meta-tool
-==========================
-
-.. code-block:: shell
-
    pip install wheel west
 
-Use ``west`` to initialize the workspace and install dependencies
-=================================================================
+Use ``west`` to initialize and install
+======================================
 
 .. code-block:: shell
 
    cd ~/golioth-reference-design-greenhouse
-   west init -m git@github.com:golioth/reference-design-greenhouse.git .
+   west init -m https://github.com/golioth/reference-design-greenhouse.git .
```

Comment:
How come you are changing the scheme?

Reply:
During the merge commit I thought the template used https. I've changed this in a commit to match the template.